source/integrate_with_govuk_pay/index.html.md.erb

```diff
@@ -76,6 +76,20 @@ You could also connect a customer-relationship management (CRM) or case manageme
 
 Read more about [reporting](/reporting/#reporting) and [refunding payments](/refunding_payments/#refunding-payments).
 
+### When you'll receive payments
+
+You'll receive the money in your bank account within 2 business days of your user completing their payment.
+
+It will take 3 business days if your user completed their payment:
+
+- after midnight UTC on a weekday
+- at the weekend
+- during a bank holiday
+
+It may take longer if the payment is below the ‘minimum payout’. For example, if you use GOV.UK Pay’s payment service provider (PSP), the minimum payout is £1. If your user completes a payment of 20p, you will not receive the money until you’ve received 80p more to reach the minimum payout.
```

Comment:
We should clarify that this is £1 net, ie after transaction fees. So a test payment of £1 would actually be 87p net and below the payout. We'd need to adjust the example accordingly (or if we think it gets too long winded, then cut the example).

Reply:
Ah, good point. It's a bit tricky to add this because we don't mention exact transaction fees in the tech docs - so I think I'll need to go with cutting the example.

I might need to rejig the text a bit though. Is it true to say "Your PSP will only transfer money to you when the total of that day's payments is above their ‘minimum payout’."?

Reply:
@miriamalice Updated in https://github.com/alphagov/pay-tech-docs/pull/374/commits/b5ed8c144e88f9b620bb9b28f0664a6be40cc253 following our chat.